src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2016,6 +2016,23 @@ public void testNewHandler() throws Exception {
         verifyWarns(checkConfig, getPath("InputIndentationNewHandler.java"), expected);
     }
 
+    @Test
+    public void testChainedMethodWithBracketOnNewLine() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+
+        checkConfig.addAttribute("arrayInitIndent", "2");
+        checkConfig.addAttribute("basicOffset", "2");
+        checkConfig.addAttribute("braceAdjustment", "0");
+        checkConfig.addAttribute("caseIndent", "2");
+        checkConfig.addAttribute("forceStrictCondition", "false");
+        checkConfig.addAttribute("lineWrappingIndentation", "4");
+        checkConfig.addAttribute("tabWidth", "2");
+        checkConfig.addAttribute("throwsIndent", "2");
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
please add one more Input file (or update Input that you introduce) to actually have cases with violations
example - https://github.com/checkstyle/checkstyle/blob/8c9a446fa86e6954f931ed4f366c36181997384c/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationAndroidStyle.java#L47

Hint: please copy code with violations from regression report, there are bunch of interestingly formatted code.